release-notes/Cube/Cube_Feature_Release_10.3/Cube_10.3.1.md

```diff
@@ -41,6 +41,16 @@ ChildrenSort="Property|property=Start,asc"
 ChildrenSort="Property|property=End,desc"
 ```
 
+#### Visio - "[GroupValue]" and "[Count]" can now be used as placeholders to adjust the value inside of dynamically grouped shapes [ID_34974]
+
+<!-- MR 10.4.0 - FR 10.3.1 -->
+
+In Visio, users can now enter the placeholder "[GroupValue]" in the *ShapeGrouping* shape to display the parameter value of the shape data *GroupBy* of the first child of the group. If the *GroupBy* value has multiple column parameters or properties, the group names will be separated by a comma.
+
+Users can also use the placeholder "[Count]" from now onwards, to display the number of children in a group of dynamically positioned shapes.
+
+Up to now, the only available placeholder that could be added in the *ShapeGrouping* shape was an asterisk ("\*").
```

Comment:
This sentence could be simplified to "Up to now, the only placeholder supported in the *ShapeGrouping* shape ..."